coordinator/internal/controller/api/get_task.go

```diff
@@ -42,32 +42,32 @@ func (ptc *GetTaskController) GetTasks(ctx *gin.Context) {
 	var getTaskParameter coordinatorType.GetTaskParameter
 	if err := ctx.ShouldBind(&getTaskParameter); err != nil {
 		nerr := fmt.Errorf("prover task parameter invalid, err:%w", err)
-		coordinatorType.RenderJSON(ctx, types.ErrCoordinatorParameterInvalidNo, nerr, nil)
+		types.RenderJSON(ctx, types.ErrCoordinatorParameterInvalidNo, nerr, nil)
```

Comment:
`types.RenderJSON` is a bit weird, `response.RenderJSON` or `types.RenderJSONResponse` would make more sense, but we can keep it as it is.

Reply:
fixed [d252709](https://github.com/scroll-tech/scroll/pull/938/commits/d2527097e0f7b9afda7a1000ef04bdac2d5547be)